src/components/v2/appDetails/appDetails.api.ts

```diff
@@ -23,32 +23,29 @@ export const deleteResource = (nodeDetails: any, appDetails: any, envId: string,
         nodeDetails.group = ''
     }
 
-    if (appDetails.appType === AppType.EXTERNAL_HELM_CHART || appDetails.deploymentAppType === DeploymentAppType.helm) {
-        const data = {
-            appId: getAppId(
-                appDetails.clusterId,
-                appDetails.namespace,
-                appDetails.deploymentAppType === DeploymentAppType.helm && appDetails.appType === AppType.DEVTRON_APP
-                    ? `${appDetails.appName}-${appDetails.environmentName}`
-                    : appDetails.appName,
-            ),
-            k8sRequest: {
-                resourceIdentifier: {
-                    groupVersionKind: {
-                        Group: nodeDetails.group,
-                        Version: nodeDetails.version,
-                        Kind: nodeDetails.kind,
-                    },
-                    namespace: nodeDetails.namespace,
-                    name: nodeDetails.name,
+    const data = {
```

Comment:
please add supporting comments for the change

Reply:
added 👍